README.md

```diff
@@ -1,5 +1,6 @@
 amz-review-scraper
 ===
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
```

Comment:
Love a repo with some nice badges like this! Looks much more professional! You should also add the Travis badge, which will also outsiders if the tests are currently passing or not on master.